SEP-0002-revised.md

```diff
@@ -0,0 +1,262 @@
+
+The primary mission of the SunPy Project is **To develop and facilitate a high-quality, easy-to-use, community-led, free and open-source solar data analysis ecosystem based on the scientific Python environment**.
+This includes but is not limited to the following tasks:
+
+* developing the sunpy core library and other affiliated packages
+* supporting a community of affiliated packages
+* managing and protect the SunPy brand and identity
+* promoting SunPy to the community
+* managing SunPy assets, i.e. GitHub organization, domain name etc.
+
+## Governance Structure
+
+The SunPy Project's governance involves the following five components:
+
+* The Community
+* The Project Members
+* The Steering Committee
+* The Advisory Board
+* The Ombudsperson
+
+The SunPy project exists for the benefit of the whole Community: users, developers, and members of the wider solar physics and scientific software community.
+The Community is the ultimate authority in how the project is managed.
+This authority is exercised through the processes outlined in this document, primarily through the Steering Committee and the Project Members.
+Whenever possible, decisions shall be approved by consensus, considering input from the entire community, rather than taken unilaterally by Project Members or the Steering Committee.
+
+### Advisory Board
+
+The primary functions of the Advisory Board are:
+
+* to advise the Steering Committee on management of the project,
+* aid in finding funding for the project to ensure the core aims of the project can be met,
+* nominate a Ombudsperson,
+* ensure that the governance of the project is running as intended.
+
+The Advisory Board is self-electing, with members appointed by a vote of the existing Advisory Board.
+
+#### Membership
+
+* The Advisory Board shall consist of at least 3 members and no more than 5 members. The Advisory Board selects how many members within this range are needed.
+* All members shall serve a maximum of a two-year term, and may be re-appointed at the end of their term.
+* Membership terms should be staggered so that approximately half the number of members will end their terms in any given year.
+* The Advisory Board may appoint members to its number up to the maximum number of members.
+* The Advisory Board should work to ensure that the composition of the Advisory Board represents a diverse cross-section of the scientific community with an interest in SunPy. This diversity should consider things such as research topic, career stage, as well as all other forms of diversity such as gender.
+* An Advisory Board member may be removed by a two-thirds majority vote of the Advisory Board or by Project Members following the same procedure as for the removal of a member of the Steering Committee.
+
+#### Meetings
+
+* The Advisory Board shall meet with the Steering Committee at least every quarter.
+* At least one week (7 days) notice of a meeting shall be provided to all eligible to attend.
```

Comment:
7 days seems a bit short. How about minimum of 14 days? Ideally these should be set well in advance, so it is easy to schedule them.